### ISSUE TITLE
[Very small issue] wrong FontAwesome icon "icon-coffee"

### ISSUE THREAD

**Issue by gadgetto** [modxcms/revolution#12676]
Using "icon-coffee" as user defined icon class for a template leads to a wrong icon (please see screenshot.

<img width="489" alt="screenshot 2015-09-26 11 51 00" src="https://cloud.githubusercontent.com/assets/876737/10116950/d4c2230a-6445-11e5-87f9-74a094514333.png">

**Comment by sottwell** [comment]
I suspect that would be an issue with fontawesome, not MODX.

**Comment by sottwell** [comment]
Ah, no, I see it's a MODX problem, in the manager/templates/default/css/index.css file it has 
```
.icon-coffee:before,.icon-css:before,.icon-htm:before,.icon-html:before,.icon-js:before,.icon-less:before,.icon-scss:before,.icon-styl:before,.icon-xml:before{content:""}
```
All of that is generated in .scss files during development, so somebody with access to all of that would need to fix it.

**Comment by sottwell** [comment]
Maybe the developer working on this really needed his coffee ;)

**Comment by Alroniks** [comment]
Formally it is not a bug, because icon used in tree for sources (javascript and sugar-language [coffeescript](http://coffeescript.org/)). But issue exists, needs to separate tree for elements and tree for sources and define different icons.

**Comment by sottwell** [comment]
I suppose it's a reference to coffeescript. Still, fa-coffee is not the source code icon.

**Comment by Mark-H** [comment]
Fixed in 2.5
